docs/concepts/comparisons.md

```diff
@@ -0,0 +1,35 @@
+---
+title: 'IPFS Comparisons'
+description: Comparing IPFS to other solutions.
+---
+
+# Storage Solution Comparisons
+
+## How is IPFS different from other solutions?
+
+IPFS is a general-purpose file system that uses a distributed hash table (DHT) to store and retrieve data. This sets it apart from other solutions with a more specific focus or using different data storage mechanisms. For example:
+
+- [BitTorrent](https://www.bittorrent.com/) is a peer-to-peer (P2P) file-sharing protocol that uses a centralized tracker to manage the distribution of files among peers. It focuses on file-sharing rather than file storage.
+- [Storj](https://storj.io/) and [Sia](https://sia.tech/) are decentralized cloud storage platforms that use distributed networks of nodes for data storage. They focus on providing cloud storage services rather than a general-purpose distributed file system.
+- [Arweave](https://www.arweave.org/) is a decentralized, permanent storage platform that uses a novel data structure called a "blockweave" for data storage. It focuses on providing permanent storage rather than a file-sharing system.
+- [Filecoin](https://filecoin.io/) is a decentralized storage network that allows users to rent out disk space. It focuses on providing a decentralized storage marketplace. It uses a proof-of-replication consensus mechanism and supports payment in various cryptocurrencies.
+  > Filecoin is built on IPFS and uses the IPFS network for data storage and retrieval. Filecoin and IPFS are complementary technologies providing decentralized and efficient storage solutions.
+- [Dat](https://datproject.org/) is a decentralized data-sharing tool that uses a distributed hash table (DHT) for data storage. It focuses on enabling data sharing and collaboration.
```

Comment:
It's more commonly referred to as the Hypercore protocol nowadays. https://hypercore-protocol.org/